index.html

```diff
@@ -20,59 +20,58 @@
   </head>
 
   <body>
-    <div class="header">
-
-        <a href="#">
-          <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
-        </a>
-
-        <div>
-          <ul class="navigation__list">
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Home</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">About Us</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">News & Events</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Contact Us</a>
-            </li>
-          </ul>
-        </div>
-
-        <button>Donate Now</button>
-
-    </div>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <nav>
+        <ul class="navigation__list">
```

Comment:
Hi, when users hover on the link, can you make the background white and make the text  color orange?
![Screenshot 2023-09-27 at 09 52 30](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/6be4f01b-147a-40a4-a043-559f8d50a442)